students/application.md

```diff
@@ -8,25 +8,139 @@ current-sub: application
 permalink: /students/application/
 ---
 
-<h1>Application Guide</h1>
+<h1>{{page.title}}</h1>
 
-Applications for Rails Girls Summer of Code will be open for 1 month <strong>starting mid February 2016</strong>.
-We're currently revamping the guide for 2016 and will update it soon. Stay tuned!
+<p>This guide gives you all the information about what you will need when applying for Summer of Code. Please read this carefully to ensure you know about all the requirements of a good and complete application.</p>
 
-In the meanwhile, you can take a look at the eligibility criteria below.  
+**Applications for Rails Girls Summer of Code will be open starting February 22nd and will close on March 15th, at 18:00 CET**.
```

Comment:
Hey @alicetragedy – I'm not very optimistic that we'll have the renewed application process ready by next monday.